packages/core/src/indexing/database-index-api.ts

```diff
@@ -81,6 +103,25 @@ export abstract class DatabaseIndexApi<DateType = Date | number> {
     this.syncApi = api
   }
 
+  /**
+   * Add a composite index to the database
+   * @param model
+   * @param indices
+   */
+  async addCompositeIndex(model: StreamID, indices: Array<ModelIndex>): Promise<void> {
```

Comment:
```suggestion
  async addModelIndex(model: StreamID, indices: Array<ModelIndex>): Promise<void> {
```

Ceramic has no notion of composites, seems confusing to use the term here